cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -174,6 +176,8 @@ public TwillRunnableSpecification configure() {
 
   @Override
   public void initialize(TwillContext context) {
+    System.setSecurityManager(new RunnableSecurityManager());
```

Comment:
No resetting. The SecurityManager is used through the execution of the container.